src/collectd.conf.pod

```diff
@@ -6625,6 +6633,29 @@ their configuration can be found in L<types.db(5)>.
 
 This optional setting sets the type instance to use.
 
+=item B<LatencyPercentile> I<Percent>
+
+Calculate and dispatch the configured percentile, i.e. compute the latency, so
+that I<Percent> of all matched latency values are smaller than or equal to the
+computed latency.
+
+Different percentiles can be calculated by setting this option several times.
+
+=item B<LatencyPercentileType> I<Type>
```

Comment:
> should use the latency type. 
> Allowing users to use a different type has next to no benefit 

One module (generic module!) - one 'plugin name', one type? Oh, no. It is not flexible enough.
In `tail`, I can use more than one <Match> per file with `DSType latency`. That will require configurable `type`.